CONTRIBUTING.md

```diff
@@ -59,6 +59,7 @@ help us to correct style violations reported here:
 
 ## Release Process
 
+* Upgrade gems with `scripts/update-gemspec`
```

Comment:
I'm not sure I agree with this being done in most / all PR's. Isolating gem updates feels best done to separate PR's.

Reply:
Indeed. Maybe we'll do it in a separate PR next time. We'll discuss that with @aslakhellesoy.